package com.revature.exceptions;

public class UserNotFoundException extends RuntimeException{

	public UserNotFoundException(String arg0) {
		super(arg0);
	}
}
